site/src/pages/UserSettingsPage/ExternalAuthPage/ExternalAuthPageView.tsx

```diff
@@ -60,7 +60,7 @@ export const ExternalAuthPageView: FC<ExternalAuthPageViewProps> = ({
           <TableHead>
             <TableRow>
               <TableCell>Application</TableCell>
-              <TableCell>Link</TableCell>
+              <TableCell />
```

Comment:
I do think that not showing the "Link" header is cleaner, and it's not a WCAG violation, but Axe is flagging this as [not being a "best practice"](https://dequeuniversity.com/rules/axe/4.8/empty-table-header?application=AxeChrome)

Three options come to mind (with the third option obviously taking much more time):
- Add the text back
- Add the text back but make it visually hidden (helps blind users, which is the biggest benefit, but it feels like we're bending the spirit of the recommendation a little)
- If we do want to cut down on UI noise, maybe we could remove the column, and make the Application cell include everything? Or maybe even refactor this from a table to a list?